interfaces/devdoc/threadpool_requirements.md

```diff
@@ -168,33 +170,33 @@ MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), thr
 MOCKABLE_FUNCTION(, int, threadpool_schedule_work, THREADPOOL_HANDLE, threadpool, THREADPOOL_WORK_FUNCTION, work_function, void*, work_function_context);
 ```
 
-`threadpool_schedule_work` schedules a work item to be executed by the threadpool.
+`threadpool_schedule_work` schedules a work item to be executed by the `threadpool`.
```

Comment:
>` [](http://example.com/codeflow?start=71&length=1)

Nitpick: Same here, no need for the backticks.

Reply:
I just wanted to make it consistent, if you see line 159 above, both `threadpool` and `non-zero` are backticked. The changes are not by me.

Reply:
Removing it at the places I made changes and keeping the threadpool object as is.

Reply:
But isn't threadpool parameter to the function `threadpool_schedule_work`?

Reply:
Done